pkg/policy/api/cidr.go

```diff
@@ -199,3 +208,18 @@ func addrsToCIDRRules(addrs []netip.Addr) []CIDRRule {
 // A CIDR Group is a list of CIDRs whose IP addresses should be considered as a
 // same entity when applying fromCIDRGroupRefs policies on incoming network traffic.
 type CIDRGroupRef string
+
+const LabelPrefixGroupName = "io.cilium.policy.cidrgroupname"
+
+func LabelForCIDRGroupRef(ref string) labels.Label {
+	var key strings.Builder
+	key.Grow(len(LabelPrefixGroupName) + len(ref))
```

Comment:
nit: Shouldn't this be

```suggestion
	key.Grow(len(LabelPrefixGroupName) + 1 + len(ref))
```

to accommodate for the `/`?

Reply:
:facepalm: yes! Good catch!